platform/wismote/dev/ext-sensor.c

```diff
@@ -43,8 +43,10 @@ static uint8_t active;
 static int
 value(int type)
 {
-  /* ADC0 corresponds to the port under the logo, ADC1 to the port over the logo,
-     ADC2 and ADC3 corresponds to port on the JCreate bottom expansion port)
+  #if 0
+  /* ADC0 corresponds to the port under the logo,
```

Comment:
The code is also not active at the moment ([platform/wismote/dev/ext-sensor.c#L44](https://github.com/contiki-os/contiki/blob/master/platform/wismote/dev/ext-sensor.c#L44)) and I changed the "disable code" style to match the rest of the file.